pkg/controller/container-runtime-config/container_runtime_config_controller_test.go

```diff
@@ -331,31 +332,49 @@ func (f *fixture) expectUpdateContainerRuntimeConfig(config *mcfgv1.ContainerRun
 	f.actions = append(f.actions, core.NewRootUpdateSubresourceAction(schema.GroupVersionResource{Version: "v1", Group: "machineconfiguration.openshift.io", Resource: "containerruntimeconfigs"}, "status", config))
 }
 
-func (f *fixture) verifyRegistriesConfigContents(t *testing.T, mcName string, imgcfg *apicfgv1.Image, icsp *apioperatorsv1alpha1.ImageContentSourcePolicy) {
+func (f *fixture) verifyRegistriesConfigAndPolicyJSONContents(t *testing.T, mcName string, imgcfg *apicfgv1.Image, icsp *apioperatorsv1alpha1.ImageContentSourcePolicy, numFiles int) {
 	icsps := []*apioperatorsv1alpha1.ImageContentSourcePolicy{}
 	if icsp != nil {
 		icsps = append(icsps, icsp)
 	}
 	updatedMC, err := f.client.MachineconfigurationV1().MachineConfigs().Get(mcName, metav1.GetOptions{})
 	require.NoError(t, err)
-	verifyRegistriesConfigContents(t, updatedMC, mcName, imgcfg, icsps)
+	verifyRegistriesConfigAndPolicyJSONContents(t, updatedMC, mcName, imgcfg, icsps, numFiles)
 }
 
-func verifyRegistriesConfigContents(t *testing.T, mc *mcfgv1.MachineConfig, mcName string, imgcfg *apicfgv1.Image, icsps []*apioperatorsv1alpha1.ImageContentSourcePolicy) {
+func verifyRegistriesConfigAndPolicyJSONContents(t *testing.T, mc *mcfgv1.MachineConfig, mcName string, imgcfg *apicfgv1.Image, icsps []*apioperatorsv1alpha1.ImageContentSourcePolicy, numFiles int) {
```

Comment:
Non-blocking I guess: The `numFiles` variable with magic 1/2 values is non-obvious; a `verifyPolicyJSON` bool would be cleaner. Or maybe split this into two/three functions, one to verify `registries.conf`, one to verify `policy.json`, one to verify the layout of `mc`, or something like that. OTOH, it’s ”only tests” in the end…

Reply:
changed to `verifyPolicyJSON` bool instead